akka-testkit-typed/src/main/scala/akka/testkit/typed/BehaviorTestkit.scala

```diff
@@ -196,7 +199,8 @@ class BehaviorTestkit[T] private (_name: String, _initialBehavior: Behavior[T])
    */
   def run(msg: T): Unit = {
```

Comment:
We could even consider having `run` return the new behavior before canonicalization? Or would that be a bit obscure?

Reply:
normally you don't care about that when testing, I think, so might be confusing to always have that in your face